app/flatpak-builtins-build-export.c

```diff
@@ -652,7 +653,7 @@ flatpak_builtin_build_export (int argc, char **argv, GCancellable *cancellable,
   guint64 installed_size = 0,download_size = 0;
   GTimeVal ts;
 
-  context = g_option_context_new (_("LOCATION DIRECTORY [BRANCH] - Create a repository from a build directory"));
+  context = g_option_context_new (_("LOCATION DIRECTORY [[COLLECTION-ID] BRANCH] - Create a repository from a build directory"));
```

Comment:
> Not sure about the order here. You might want to have a branch, but not set a collection id, whereas there is always a branchname, even if you leave the argument out.

Those cases can be differentiated by looking at `argc`, but I agree that it’s a bit confusing. I put the collection ID before the branch name because it’s always presented before the branch name elsewhere — i.e. like in the tuple `($collection_id, $ref_name)`.

> In fact, isn't it better if this is --collection-id=ID

I decided against that because ideally most flatpaks are (eventually) going to have a collection ID set, so it would ideally be a positional argument. However, `--collection-id` is probably a better solution to prevent the weirdness with `argc` and ordering. I’ll change it.